src/main/java/org/openhds/service/contract/AbstractUuidService.java

```diff
@@ -96,6 +96,16 @@ public T findOne(String id) {
         return repository.findOne(id);
     }
 
+    public T findOrMakePlaceHolder(String uuid){
+        T entity = findOne(uuid);
+        if (null == entity){
+            entity = getUnknownEntity();
```

Comment:
Maybe this should use makeUnknown instead of getUnknown.

I think the difference is that makeUnknown will create a new one, and the new one will have its own insert date and user.  I think getUnknown will cause all placeholders to have the same insert date and user.